json-format.md

```diff
@@ -129,34 +129,82 @@ OPTIONAL not omitted attributes MAY be represeted as a `null` JSON value.
 
 ### 3.1. Handling of "data"
 
+The JSON representation of the event "data" payload is determined by the runtime
```

Comment:
I think it's necessary as well. The existing language below:

> If the implementation determines that the type of data is `Binary`...

already implies runtime type, otherwise what does "type of data" even mean?